Scripts/Services/LootGeneration/RunicReforging/RunicReforging.cs

```diff
@@ -941,30 +941,12 @@ public static bool ApplyResistance(Item item, int value, AosElementAttribute att
 
         public static int Scale(int min, int max, int perclow, int perchigh, int luckchance, bool reforged)
         {
-            int percent;
-
-            if (reforged)
-            {
-                percent = Utility.RandomMinMax(perclow, perchigh);
-            }
-            else
-            {
-                percent = Utility.RandomMinMax(0, perchigh);
-            }
-
+            decimal percent = Utility.RandomMinMax(perclow, perchigh);
             if (LootPack.CheckLuck(luckchance))
                 percent += 10;
+            percent = percent > perchigh ? perchigh : percent;
 
-            if (percent < perclow) percent = perclow;
-            if (percent > perchigh) percent = perchigh;
-
-            int scaledBy = Math.Abs(min - max) + 1;
-
-            scaledBy = 10000 / scaledBy;
-
-            percent *= 10000 + scaledBy;
-
-            return min + (max - min) * percent / 1000001;
+            return Convert.ToInt32(Math.Round(((max - min) * percent / 100) + min));
```

Comment:
Same calculation as previous, but in less code

Reply:
This is great. Anything that makes it easier to read / less code.